docs/topics/install/docker.rst

```diff
@@ -140,6 +135,81 @@ migrations::
 Gotchas!
 ~~~~~~~~
 
+Here's a list of a few of the issues you might face when using docker.
+
+Can't access the web server?
+----------------------------
+
+Check you've created an hosts file entry pointing ``olympia.dev`` to the
+relevant ip address.
+
+Also make sure you've copied ``docker-compose.override.yml.tmpl`` to
+``docker-compose.override.yml`` to get the default ports. If you haven't
+stop the containers with ``docker-compose stop`` copy the file and restart
+with ``docker-compose up -d``.
+
+Another tip is to use ``docker-compose ps`` to check the status of the
+containers. If they are failing to start you should be able to tell here.
+
+Another way to find out what's wrong is to run ``docker-compose logs``.
+
+Getting "Programming error [table] doesn't exist"?
+--------------------------------------------------
+
+Check you've run the ``make initialize_docker`` step.
+
+
+Port collisions (nginx container fails to start)
+------------------------------------------------
+
+Since by default the docker-compose file exposes the port to the nginx
+server that sits in front of the web service on port 80 on your host
+you might find it fails to start if you're already running a service on
+port 80.
+
+This problem will manifest itself by the services failing to start, you'll
+be able to see the error like so::
+
+    ERROR: for nginx  Cannot start service nginx:.....
+    ...Error starting userland proxy: Bind for 0.0.0.0:80: unexpected error (Failure EADDRINUSE)
+    ERROR: Encountered errors while bringing up the project.
+
+There's a couple of ways to fix it. Simple one is to find out what's running on
```

Comment:
For some people (maybe who have a MAMP or something similar installed that already runs on port 80), this might not be the simple way to open up port 80, and might only work until a restart (when whatever was running there will start again). Plus giving a `sudo` command as the simple solution (especially when they'll likely need to `sudo kill` a process running on port 80) seems bad.

Giving them a way to change the port first seems best.